pkg/codeowners/codeowners.go

```diff
@@ -0,0 +1,46 @@
+package codeowners
+
+import (
+	"os"
+
+	"github.com/hairyhenderson/go-codeowners"
+)
+
+type Codeowners struct {
+	codeowners *codeowners.Codeowners
+}
+
+func NewCodeowners(path string) (*Codeowners, error) {
+	// open filesystem rooted at current working directory
+	fsys := os.DirFS(path)
+
+	c, err := codeowners.FromFileWithFS(fsys, "CODEOWNERS")
+	if err != nil {
+		return nil, err
+	}
+
+	return &Codeowners{
+		codeowners: c,
+	}, nil
+}
+
+func (c *Codeowners) GetOwners(path string) (string, bool) {
+	if c == nil || c.codeowners == nil {
+		return "", false
+	}
+
+	owners := c.codeowners.Owners(path)
+	if len(owners) == 0 {
+		return "", false
+	}
+
+	// TODO: load ownermap from standard-insights local depoyment file
+	ownerMap := map[string]string{}
```

Comment:
Why we cannot use directly what's inside the owners variable and why do we need an additional mapping ?

Reply:
I agree it would be better to only rely on what is inside CODEOWNERS, but some CODEOWNERS file might have slightly different namings compared to what is needed in the emitted metrics labels and dashboards

Reply:
I will add a dedicated top level configuration for this